pulpcore/cli/core/task.py

```diff
@@ -25,7 +25,14 @@ def task(ctx: click.Context, pulp_ctx: PulpContext) -> None:
                 "name__contains",
                 help=_("List only tasks whose name contains this."),
             ),
-            click.option("--state", help=_("List only tasks in this state.")),
+            click.option(
+                "--state",
+                type=click.Choice(
+                    ["waiting", "skipped", "running", "completed", "failed", "canceled"],
+                    case_sensitive=False,
```

Comment:
how does that work?
Is "wAiTiNg" transformed into "waiting" automatically?

Reply:
Yes, as far as I understand (just tested with a minimal example), setting `case_sensitive` to `False` causes the value passed to be normalized to lowercase.

Reply:
Can you add too calls to the tests to show this off?
`--state FinisheD` and `--state cannotwork` for example...